config/default/webp.settings.yml

```diff
@@ -1,3 +1,3 @@
 _core:
   default_config_hash: 0ube66wBYymHixNiMD-uyY8Tjo8alYV3NII6dv5iBJ8
-quality: 85
+quality: 80
```

Comment:
Note: These compression changes (although minor) contribute to the performance gains achieved in the linked performance tests. These were ported over from a previously rejected PR.